### DISCUSSION TITLE
[Feature Request] Having color codes for each theme/notes groupe.

### DISCUSSION THREAD

**Discussion by Dreykku** [standardnotes/forum#2618] [Suggestions]
Hi,

First time using Standardnotes. Free for now, seriously considering getting the 5years plan. It seems to have it all but something is really bothering me. 
Apparently there is no color code in front of the note names. Like most "to-do" apps have (like [todoist](https://i0.wp.com/www.dandywithlens.com/wp-content/uploads/2018/03/Todoist-projects-e1520538606805.gif?fit=800%2C552) for example). I see only small circles of the same colors (except for subnotes). In the free version we don't even have those circles, only "#" from tags. I guess it comes with the premium version? 

I edited one of your demo to show what i'm talking about : https://imgur.com/a/88iSSQr
It would increase the workflow so considerably just by navigating from color to color instead of having to read the titles of our notes. For people who have many many categories of notes, it's a lot of time wasted trying to read our own notes groupe titles and select the right one. And after awhile this time wasted adds up quite considerably. Those short confusing moments could be resolved just by adding a color system. We could even imagine other geometrical forms instead of circles like triangles, square etc (+ the color system) to increase the workflow even more. 
And even small emojis at some points, like in some popular "to-do" apps (a plate for "food" theme, a dumbbell for "sport", etc). It's a bonus but it could nice also. 

As a side note, it's not a very user friendly software i must say, compared to its popular alternatives. It could use some simple guides for the casuals to introduce us more globally to the software before we dwell on the more detailed guides (for example it took me some time to understand i couldn't make clickable links in the free version because of the limited plain editor, that's not obvious to everyone).  And i still don't know how to actually make those circles yet, colored or not. 

Posted in [reddit ](https://www.reddit.com/r/StandardNotes/comments/g68toc/feature_request_having_color_codes_for_each/) as well. 

Thank you.

**Comment by JaspalSuri** [comment]
Hi @Dreykku,

Welcome to Standard Notes and thank you for considering supporting us by subscribing to the 5-year plan! 

I've taken note of your feedback regarding color coded tags and notes, and the ability to change the circles into other shapes or replace them with emojis. 

One thing you could do now to visually distinguish tags and notes is to insert an emoji into the tag's name and insert an emoji at the beginning of each title. 

> As a side note, it's not a very user friendly software i must say, compared to its popular alternatives. It could use some simple guides for the casuals to introduce us more globally to the software before we dwell on the more detailed guides (for example it took me some time to understand i couldn't make clickable links in the free version because of the limited plain editor, that's not obvious to everyone). 

I'm sorry to hear that. I'll let the team know about this so more guides (and maybe even videos) are available. That said, we are working on improving the design of the app so it's more intuitive for everyone. :) 

> And i still don't know how to actually make those circles yet, colored or not.

Click on the Extensions button near the bottom left corner, scroll down until you see the Folders extension, click "install", and then click “activate”. This should enable that feature for you! Clicking on the circle will collapse the list of tags beneath the folder tag. Check it out in the [demo editor here](https://standardnotes.org/extensions/folders).

[Here's a video](https://imgur.com/a/s3E4XpW) of how to do this. 

Once you have it installed, it will be available under the "Components" section unless you uninstall it.

> Posted in reddit as well.

Thanks, I'll respond there as well.

**Comment by Dreykku** [comment]
Thank you for considering my request. 

> Click on the Extensions button near the bottom left corner, scroll down until you see the Folders extension, click "install", and then click “activate”. This should enable that feature for you! Clicking on the circle will collapse the list of tags beneath the folder tag. Check it out in the demo editor here.
> 
> Here's a video of how to do this.
> 
> Once you have it installed, it will be available under the "Components" section unless you uninstall it.

As i stated, I have the free version right now. I've been using Standardnotes for barely 2 days. You're confirming me that i have to get the premium version to get the circles instead of the hashtags. And if i get the premium version, only the 5 years plan is worth it, which doesn't leave me much choice between nothing and 5 years commitment. 

This is another issue i find. There is no way to fully understand what the premium version can offer without trying it going from free to premium. Even though we have the demos, we don't know exactly what's a feature from an extension, or several extensions combined within one demo, and what is a feature within the free software that we don't know how to use yet. 
A free trial of 15 days for example, without having to pay in advance to then get a refund if not satisfied, would solve that issue. We would know what's what and then decide to buy the premium version or not.

**Comment by JaspalSuri** [comment]
> As i stated, I have the free version right now. I've been using Standard Notes for barely 2 days. 

I apologize for not taking that important detail into account. I must have had an off-day yesterday. 

>  And if i get the premium version, only the 5 years plan is worth it, which doesn't leave me much choice between nothing and 5 years commitment.

That depends on how you see it; you can try the app for a year as that will give you plenty of time to decide if it has worked well for you. From there you can subscribe to the five-year plan if you thoroughly enjoy using it. 

> This is another issue i find. There is no way to fully understand what the premium version can offer without trying it going from free to premium. Even though we have the demos, we don't know exactly what's a feature from an extension, or several extensions combined within one demo, and what is a feature within the free software that we don't know how to use yet.

You can take a look at our [extensions page](https://standardnotes.org/extensions) to learn more about the different features that they offer and then use a demo to see if the different editors work well enough in your personal use-cases.

> A free trial of 15 days for example, without having to pay in advance to then get a refund if not satisfied, would solve that issue. We would know what's what and then decide to buy the premium version or not.

Currently we offer a 30 day trial that does require payment up-front but we also have a no-questions-asked refund policy if a timely request is made. I'll be sure to provide the team with your feedback on offering other free trial options.

**Comment by Dreykku** [comment]
Addendum to my request : 
In the same spirit of workflow increase and improvement of clarity, also have the circles -with the corresponding colors- instead of the hashtags not only on the list on the left but also on the notes list in the middle and even under the title at the top of the note. Basically, replace all the hashtags from the tags by the corresponding colored circles. Or at least make it an option to choose to activate it or not.
That would give the notes a certain "color tonality" depending on the theme we are on, the corresponding dots coloring almost all parts of the UI no matter where we're looking at (under the title, at the note list, or at tags list)
Example : 
https://imgur.com/a/NTubJSH (I made subtags within the same tag of same color cause it's not as important, but even more refined color codes within the same tag could be nice also)



> Currently we offer a 30 day trial that does require payment up-front but we also have a no-questions-asked refund policy if a timely request is made. I'll be sure to provide the team with your feedback on offering other free trial options.

I think there is a reason why the "free trial" business model for softwares has been so popular. It works. 
To my mind, it's much more efficient to have users experiencing the full potential of the software freely to then realize 15 or 30 days later "Hey, that option was very useful, I don't have it anymore with the free version? I guess it's worth paying for it then", rather than having people "guessing" for what they could get if they ever take the premium version even if they theoretically know it on paper, they still have to make the effort to do some research on it. I don't think it's quite as effective. 

Thank you for your listening.

**Comment by JaspalSuri** [comment]
Thanks for the addendum and for providing another illustration of the concept! 

> I think there is a reason why the "free trial" business model for softwares has been so popular. It works. To my mind, it's much more efficient to have users experiencing the full potential of the software freely to then realize 15 or 30 days later "Hey, that option was very useful, I don't have it anymore with the free version? I guess it's worth paying for it then", rather than having people "guessing" for what they could get if they ever take the premium version even if they theoretically know it on paper, they still have to make the effort to do some research on it. I don't think it's quite as effective.

I'll take note of your feedback for this! It certainly wouldn't hurt if there was a better way to have more users support us with a subscription. :)
